philu_actions/scripts/dependency_downloader.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -xe
+
+SHORT_DIST="xenial" # Update this if you update ubuntu version
+
+# Bootstrapping constants
+PIP_VERSION="9.0.3"
+EDX_PPA_KEY_SERVER="keyserver.ubuntu.com"
+EDX_PPA_KEY_ID="B41E5E3969464050"
```

Comment:
Is this something public?

Reply:
Yes, it is taken from [edx ansible script](https://github.com/edx/configuration/blob/08800400716e918f9b08c6ee05e24f734a798104/util/install/ansible-bootstrap.sh#L47rL48)